composer.json

```diff
@@ -54,4 +54,4 @@
             ]
         }
     }
-}
+}
```

Comment:
All files should end with an empty newline. Could you fix this for all your PRs?